python/iterating_for_each_two_elements.md

```diff
@@ -0,0 +1,28 @@
+# Interating a list for each 2 elements using `zip` and `iter`
```

Comment:
Perhaps there is a better way to phrase this (as we're not iterating over the resulting list)? "Converting a list into pairs of elements?"

What's the equivalent in Ruby? `each_slice(2)`?